deb/openmediavault/usr/share/openmediavault/engined/rpc/diskmgmt.inc

```diff
@@ -90,8 +90,9 @@ class DiskMgmt extends \OMV\Rpc\ServiceAbstract {
 				"serialnumber" => $sd->getSerialNumber(),
 				"wwn" => $sd->getWorldWideName(),
 				"israid" => $sd->isRaid(),
-				"isroot" => \OMV\System\System::isRootDeviceFile(
-					$sd->getCanonicalDeviceFile(), FALSE)
+				"isroot" =>
+				$rootDeviceFile === basename($sd->getCanonicalDeviceFile()) ?
```

Comment:
Why using basename here?

Reply:
Because `getBlockParentDevice ` returns only device name to `getRootDeviceFile `, therefore `getRootDeviceFile` will also returns device name without path. We compare only names.